crt_portal/static/js/word_count.js

```diff
@@ -30,10 +31,10 @@ function updateWordCount (e) {
       wordLimitAlert.removeAttribute('hidden');
       wordLimitAlert.setAttribute('role', 'alert');
     } else {
-      var wordsRemaining = 500 - words;
+      var wordsRemaining = max - words;
       displayCountElem.innerHTML = wordsRemaining;
       countMessageElem.value = (wordsRemaining === 1) ? 'word remaining' : 'words remaining';
-      textAreaElem.classList.remove('bg-gold-outline');
+      // textAreaElem.classList.remove('bg-gold-outline');
```

Comment:
It looks like this line was commented out by accident — removing this introduces a bug with the text area alert state border, where once the border is added it stays permanently.

Reply:
Great catch